test/gtest/Makefile.am

```diff
@@ -70,14 +70,16 @@ gtest_LDADD = \
 	$(OPENMP_CFLAGS) \
 	$(GTEST_LIBS)
 
-
-gtest_CPPFLAGS = \
-	$(BASE_CPPFLAGS) \
+gtest_basic_CPPFLAGS = \
```

Comment:
IMO the cuda code should have only `-I$(top_srcdir)/src` so no need for gtest_basic_CPPFLAGS var